src/react.spec.ts

```diff
@@ -3,7 +3,8 @@ import { createElement, ReactComponent, reactDOMRender } from "./react";
 describe('React', () => {
     let body: Element;
     beforeEach(() => {
-       body = document.body;
+        document.body.innerHTML = ''
```

Comment:
Cleaning up the existing markup is a good idea, but I don't think it belongs in tests.
Render should be able to handle a non-empty element. Think about it as a re-render:
```
reactDOMRender(createElement("div", {}));
reactDOMRender(createElement("span", {}));
reactDOMRender(createElement("div", { children: "hey" }));
```

Code above could be used by whoever uses our fake react.
So the render should be able to handle that situation.

What I'm leading to, is that this cleanup belongs in `reactDOMRender`